### PR TITLE
feat: add Coding a Page guide

### PR DESCRIPTION
**Pull request by wp-aberg**
This PR adds the Coding a Page guide that builds out a recipe search page step by step

### CODE REVIEW

File: docs/resources/guides/coding-a-page.mdx [hunk 1]
```diff
@@ -20,22 +25,870 @@ You will finish this tutorial with a strong opinion from us on how to best devel
...
+
+### Structural Elements
+
+```jsx
+import React from "react";
+import { Container, styled } from "@washingtonpost/wpds-ui-kit";
+
+const StyledContainer = styled(Container, {
+  alignItems: "unset",
+  backgroundColor: "$blue600",
```
Comment (artmsilva) [inline docs/resources/guides/coding-a-page.mdx]: I think we should use theme object here and elsewhere in the guide in order to encourage our best practice as a community best practice.

File: docs/resources/guides/coding-a-page.mdx [hunk 2]
```diff
@@ -20,22 +25,870 @@ You will finish this tutorial with a strong opinion from us on how to best devel
...
+  minHeight: "100vh",
+});
+
+export default function Home() {
+  return <StyledContainer>Container</StyledContainer>;
+}
+```
+
+WPDS ships with a responsive Container component that centers its content and resizes based on window width. It has four sizes based on UI Kit's internal breakpoints. For small and medium breakpoints its width is 100%, large its width is 900px, extra large and extra extra large 1024px, and above extra large 1440px. In the initial stage of developing the layout we'll use the [stitches `styled` function](https://stitches.dev/docs/api#styled) that ships with UI Kit to make the container easier to visualize.
```
Comment (atomictangerine) [inline docs/resources/guides/coding-a-page.mdx]: nit: For small and medium breakpoints the container's width is 100%, large is 900px, extra large and extra extra large is 1024px, and above extra large is 1440px.

File: docs/resources/guides/coding-a-page.mdx (hunk 1 above)
Comment (atomictangerine) [reply docs/resources/guides/coding-a-page.mdx]: +1

File: docs/resources/guides/coding-a-page.mdx [hunk 3]
```diff
@@ -20,22 +25,870 @@ You will finish this tutorial with a strong opinion from us on how to best devel
...
+### Structural Elements
+
+```jsx
+import React from "react";
+import { Container, styled } from "@washingtonpost/wpds-ui-kit";
+
+const StyledContainer = styled(Container, {
+  alignItems: "unset",
+  backgroundColor: "$blue600",
+  gap: "$100",
```
Comment (atomictangerine) [inline docs/resources/guides/coding-a-page.mdx]: we should probably also use the `theme.sizes['100']` token structure as that's best practice for using spacing tokens

File: docs/resources/guides/coding-a-page.mdx [hunk 4]
```diff
@@ -20,22 +25,870 @@ You will finish this tutorial with a strong opinion from us on how to best devel
...
+  <InlineImage
+    width={900}
+    height={610}
+    maxWidth="100%"
+    src="/img/guides/coding-a-page/layout_01.png"
+  />
+</Container>
+<br />
+
+By default Container is set to display flex with a direction of column, our layout adds two children. A Header to contain the elements that span the width of the page, and a OverviewDetail to contain the grid of recipe photos and the selected recipe's details. StyledContainer un-sets alignItems to allow its child elements to have a 100% width. It also sets a flex property of 1 on OverviewDetail using an nth-child selector to allow it to expand and fill any vertical space not occupied by the Header. Using a child selector keeps OverviewDetail separated from the concern of how it is layed out in its parent.
```
Comment (atomictangerine) [inline docs/resources/guides/coding-a-page.mdx]: nit: A Header component contains elements that span the width of the page and an OverviewDetail component contains the grid of recipe photos and the selected recipe's details.

Comment (atomictangerine) [reply docs/resources/guides/coding-a-page.mdx]: nit: By default, the Container component is set to `display:flex` with a direction of column, our layout adds two children.  <= This sentence is a little confusing. Possibly reword?

**Review by atomictangerine — COMMENTED** [review]
so cool! Great job on this! Also, love the extensive docs.

File: docs/resources/guides/coding-a-page.mdx (hunk 1 above)
Comment (wp-aberg) [reply docs/resources/guides/coding-a-page.mdx]: Updated in 8ea287d

File: docs/resources/guides/coding-a-page.mdx (hunk 3 above)
Comment (wp-aberg) [reply docs/resources/guides/coding-a-page.mdx]: Updated in 8ea287d

File: docs/resources/guides/coding-a-page.mdx (hunk 2 above)
Comment (wp-aberg) [reply docs/resources/guides/coding-a-page.mdx]: changed in b32ecd8

File: docs/resources/guides/coding-a-page.mdx (hunk 4 above)
Comment (wp-aberg) [reply docs/resources/guides/coding-a-page.mdx]: Went ahead and broke it into two sentences.

changes in b32ecd8

**Review by artmsilva — APPROVED** [review]

**Pull request merged**